assets/cms/js/vue/Manager.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-new */
+/* global Vue */
+import Context from './Context'
+
+class Manager {
+    /**
+     * Ensures that our Concrete.Vue manager is available on the window object.
+     * Note: Do NOT call this before the global Concrete object is created in the CMS context.
+     *
+     * @param window
+     */
+    static bindToWindow(window) {
+        if (!window.Concrete) {
+            window.Concrete = {}
+        }
+        if (!window.Concrete.Vue) {
+            window.Concrete.Vue = new Manager()
+            window.Concrete.Vue.contexts = []
+        }
+    }
+
+    /**
+     * Returns the Vue Context object for the current string `context`
+     *
+     * @param context
+     * @returns {*}
+     */
+    getContext(context) {
+        return window.Concrete.Vue.contexts.filter(obj => obj.context === context)[0]
+    }
+
+    /**
+     * Actives a particular context (and its components) for a particular selector.
+     *
+     * @param context
+     * @param selector
+     * @param parent
+     */
+    activateContext(context, selector, parent) {
+        var contextObject = this.getContext(context)
+        var selectors = parent.querySelectorAll(selector)
```

Comment:
You can just pass the string selector to the new Vue call, you don't need to pass an element https://vuejs.org/v2/api/#el